chart/templates/triggerer/triggerer-deployment.yaml

```diff
@@ -44,6 +45,9 @@ metadata:
     {{- toYaml .Values.triggerer.annotations | nindent 4 }}
   {{- end }}
 spec:
+  {{- if $persistence }}
+  serviceName: {{ .Release.Name }}-triggerer
+  {{- end }}
```

Comment:
I wonder if this specifically also needs to be behind a version flag?

Reply:
Can you give it a quick test?

Reply:
you know... what do you think about just keeping it simple with just the 2.2.0 flag?  It's not really gonna hurt anything to have the service there for < 2.6 and it's one less bit of logic in the chart